# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../madwifi/no-depmod.patch
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- madwifi-0.9.3/Makefile.vanilla	2007-03-23 08:39:26.000000000 +0000
+++ madwifi-0.9.3/Makefile	2007-03-23 08:39:35.000000000 +0000
@@ -93,9 +93,6 @@
 	for i in $(DIRS_MODULES); do \
 		$(MAKE) -C $$i install || exit 1; \
 	done
-ifeq ($(DESTDIR),)
-	(export KMODPATH=$(KMODPATH); /sbin/depmod -ae $(KERNELRELEASE))
-endif
 
 install-tools:
 	$(MAKE) -C $(TOOLS) install || exit 1
